test/lib.spec.js

```diff
@@ -882,7 +884,7 @@ describe("api compatibility", () => {
       hasher: sha256,
     })
 
-    const { cid, bytes } = await UCAN.write(ucan)
+    const { cid, bytes } = await UCAN.write(ucan, {hasher: sha256})
```

Comment:
i had to change the function signature/types a bit because of a new ts version. No need to add another test we would be just testing multiformat itself or a random function call.